using-valtimo/case/case-widgets/case-widgets.md

```diff
@@ -1,6 +1,11 @@
-# Case widgets
+# Case Widgets
 
-Case widgets allow for reviewing the current state of a case at a glance. To use them, a widget configuration can be
-made for a case tab.
+Under the tab details for a widgets tab you can find this overview detailing which widgets are configured so far:
 
-- [Creating a case widgets tab](creating-case-widgets-tab.md)
+![Widgets list](../img/widget-list.png)
+
+Here the order of the widgets can be change through drag and drop, widgets can be deleted or edited.
```

Comment:
Here the order of the widgets can be changed through drag and drop. Widgets can be deleted or edited.